src/sentry/dynamic_sampling/rules/base.py

```diff
@@ -37,6 +55,11 @@ def get_guarded_blended_sample_rate(organization: Organization, project: Project
     if sample_rate == 1.0:
         return float(sample_rate)
 
+    # In case the organization or the project have been recently added, we want to boost to 100% in order to give users
+    # a better experience. Once this condition will become False, the dynamic sampling systems will kick in.
```

Comment:
Just out of curiosity, could we handle this by introducing a "Boost new organization" rule

Reply:
This could be done yes but a new rule is not really needed, since the value will be static and we want to treat it as a base sample rate. In addition, if we opt to create a new rule, we will have to give it a type of `sampleRate` and put it on top of the uniform rule, which will make the system unnecessarily complicated.